backend/src/layer/logic/mealplan_management/test/mod.rs

```diff
@@ -0,0 +1,2 @@
+pub mod meal_plan_parser_mock;
```

Comment:
maybe you can mark, that the entire test package only gets compiled on test (with `#![cfg(test)]`)?

Reply:
added #![cfg(test)] to test/mod.rs